includes/import-products.php

```diff
@@ -139,10 +142,30 @@ function dw_add_woocommerce_product($course) {
 		return ($startA < $startB) ? -1 : 1;
 
 	});
+
+	dw_format_and_save_dates($product, $startDate, $courseParts);
+
+	update_post_meta($product->get_id(), '_product_attributes', $attributes);
+
+	return $product;
+}
+
+/**
+ * @param WC_Product $product
+ * @param DateTime $date
+ * @param array $courseParts
+ *
+ * @throws Exception
+ */
+function dw_format_and_save_dates(WC_Product $product, DateTime $date, array $courseParts) {
+	wp_set_object_terms($product->get_id(), $date->format('d-m-Y'), 'pa_datum', false);
```

Comment:
@Dirkhuethorst, if `'d-m-Y'` on this line, `'H:i'` on the next line, etc. are the exact same as declared in the `dw_add_woocommerce_product` function for `pa_datum`, `pa_tijd`, etc. then should we make a constant for them?